src/main.py

```diff
@@ -0,0 +1,102 @@
+from AlertsAggregator import AlertsAggregator
+from OrefAlertsRetrieve import OrefAlertsRetrieveData  
+from fastapi import FastAPI, HTTPException 
+import pandas as pd
+import requests
+from Errors import RetrieveDataException, WrongSettlementException
+from models import AlertsQueryInput
+from datetime import datetime, timedelta
+app = FastAPI()
+
+CSV_FILE_PATH = "C:/Projects/Python/Projects/BestTimeToShower/data/data.csv"
+dataframe = pd.read_csv(CSV_FILE_PATH)
+getdata = OrefAlertsRetrieveData()
+alert_aggregator = AlertsAggregator(dataframe)
+
+
+
+@app.post("/orefalerts")
+async def get_oref_alert(from_date: str, to_date: str):
+    try:
+        global alert_aggregator
```

Comment:
I want to make this function internal.
So only those who have an API-Key can call this function.

ask chatGPT:
"how to read an API-KEY from HTTP headers in fastapi @post function" 

invent a key of your choice and and only allow those with the key to call this function